### PR TITLE
Prism plugin bugfix alt

### PR DESCRIPTION
**Pull request by bvaughn**
Alternate option for #1488, also resolves issue #1486

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 4e09661f8d13fb14d04e0801efe7269a30913bcd

https://app.netlify.com/sites/using-styled-components/deploys/5966db607960b145d00e71cb

**Comment by bvaughn** [comment]
Look at @KyleAMathews, being mr negativity there. @gatsbybot says "preview ready!" but Kyle says "preview failed!" 🤡

File: packages/gatsby-remark-prismjs/README.md [hunk 1]
```diff
@@ -16,7 +16,16 @@ plugins: [
     resolve: `gatsby-transformer-remark`,
     options: {
       plugins: [
-        `gatsby-remark-prismjs`,
+        {
+          resolve: `gatsby-remark-prismjs`,
+          options: {
+            // Class prefix for <pre> tags containing syntax highlighting;
+            // Defaults to 'language-'.
+            // If you use Prism directly within your site,
```
Comment (KyleAMathews) [inline packages/gatsby-remark-prismjs/README.md]: Perhaps rewrite this sentence slightly so makes clear this option is only needed when loading prismjs into the browser and this is not very common.

**Comment by KyleAMathews** [comment]
I'd vote for this option as not a lot of people know how to write css targeting data attributes (I'd have to look it up) where modifying the class is a simple find/replace.

**Comment by bvaughn** [comment]
> I'd vote for this option as not a lot of people know how to write css targeting data attributes (I'd have to look it up) where modifying the class is a simple find/replace.

Sure. Sounds reasonable. My only concern with this was the similarity (in API) to Prism's own custom-class approach (which has a different implementation/impact).

I'll tidy up the comment momentarily.

Not sure what's up with CI. Looks like an NPM mirror was down or something.

**Comment by bvaughn** [comment]
Okay! Comments updated. Tested once more locally to confirm the fix. 😄  Let me know if there's anything else needing to be done.

**Comment by KyleAMathews** [comment]
Sweet! Merging and will be making a new release shortly.

>Not sure what's up with CI. Looks like an NPM mirror was down or something.

it's remarkable how often CI and building gatsby example sites fail due to random cloud outages. We're building on a castle of cards.

**Comment by jlengstorf** [comment]
Hiya @bvaughn! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Comment by bvaughn** [comment]
That's a nice gesture. Thanks, Jason.

**Pull request merged**